mycroft/skills/skill_manager.py

```diff
@@ -279,8 +279,9 @@ def handle_paired(self, _):
 
     def load_plugin_skills(self):
         plugins = find_skill_plugins()
+        loaded_skill_ids = [p.split("/")[-1] for p in self.skill_loaders]
         for skill_id, plug in plugins.items():
-            if skill_id not in self.plugin_skills:
+            if skill_id not in self.plugin_skills and skill_id not in loaded_skill_ids:
```

Comment:
Should we log something when a plugin is ignored in favor of a local clone?

Reply:
might be a good idea, but this is used in a loop to check for new plugins so logs would be very spammy... might need some refactoring

Reply:
Makes sense.. Maybe somewhere in skill load we could check for a plugin version and warn

Reply:
keep track throughout the loop, and log yes/no normally or a list of overridden plugins when log_level.DEBUG?

Reply:
```
2021-11-04 20:15:04.901 - OVOS - mycroft.skills.skill_manager:_load_on_startup:392 - INFO - Loading installed skills...
2021-11-04 20:15:04.906 - OVOS - mycroft.skills.skill_manager:_load_new_skills:420 - INFO - mycroft-hello-world.mycroftai plugin will be replaced by a local version: /home/user/.local/share/mycroft/skills/mycroft-hello-world.mycroftai
2021-11-04 20:15:04.907 - OVOS - mycroft.skills.skill_manager:_unload_plugin_skill:514 - INFO - Unloading plugin skill: mycroft-hello-world.mycroftai
2021-11-04 20:15:04.909 - OVOS - mycroft.skills.skill_loader:load:188 - INFO - ATTEMPTING TO LOAD SKILL: mycroft-hello-world.mycroftai
```